examples/local/local-container-mnist.ipynb

```diff
@@ -0,0 +1,398 @@
+      "Requirement already satisfied: setuptools>=40.8.0 in /opt/conda/lib/python3.11/site-packages (from triton==3.3.1->torch==2.7.1->torchvision) (75.8.2)\n",
+      "Requirement already satisfied: mpmath<1.4,>=1.1.0 in /opt/conda/lib/python3.11/site-packages (from sympy>=1.13.3->torch==2.7.1->torchvision) (1.3.0)\n",
+      "Requirement already satisfied: MarkupSafe>=2.0 in /opt/conda/lib/python3.11/site-packages (from jinja2->torch==2.7.1->torchvision) (3.0.2)\n",
+      "WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.\n",
+      "100%|██████████| 9.91M/9.91M [00:01<00:00, 5.53MB/s]\n",
+      "100%|██████████| 28.9k/28.9k [00:00<00:00, 233kB/s]\n",
+      "100%|██████████| 1.65M/1.65M [00:00<00:00, 2.52MB/s]\n",
+      "100%|██████████| 4.54k/4.54k [00:00<00:00, 3.37MB/s]\n",
+      "Train Epoch: 1 [0/60000 (0%)]\tLoss: 2.313989\n",
+      "Train Epoch: 1 [6400/60000 (11%)]\tLoss: 0.543117\n",
+      "Train Epoch: 1 [12800/60000 (21%)]\tLoss: 0.163005\n",
+      "Train Epoch: 1 [19200/60000 (32%)]\tLoss: 0.146137\n",
+      "Train Epoch: 1 [25600/60000 (43%)]\tLoss: 0.095676\n",
+      "Train Epoch: 1 [32000/60000 (53%)]\tLoss: 0.209810\n",
+      "Train Epoch: 1 [38400/60000 (64%)]\tLoss: 0.089851\n",
+      "Train Epoch: 1 [44800/60000 (75%)]\tLoss: 0.241108\n",
+      "Train Epoch: 1 [51200/60000 (85%)]\tLoss: 0.101316\n",
+      "Train Epoch: 1 [57600/60000 (96%)]\tLoss: 0.196654\n",
+      "Train Epoch: 2 [0/60000 (0%)]\tLoss: 0.077407\n",
+      "Train Epoch: 2 [6400/60000 (11%)]\tLoss: 0.132044\n",
+      "Train Epoch: 2 [12800/60000 (21%)]\tLoss: 0.109268\n",
+      "Train Epoch: 2 [19200/60000 (32%)]\tLoss: 0.097897\n",
+      "Train Epoch: 2 [25600/60000 (43%)]\tLoss: 0.137971\n",
+      "Train Epoch: 2 [32000/60000 (53%)]\tLoss: 0.024578\n",
+      "Train Epoch: 2 [38400/60000 (64%)]\tLoss: 0.045029\n",
+      "Train Epoch: 2 [44800/60000 (75%)]\tLoss: 0.028825\n",
+      "Train Epoch: 2 [51200/60000 (85%)]\tLoss: 0.032730\n",
+      "Train Epoch: 2 [57600/60000 (96%)]\tLoss: 0.027846\n",
+      "Training is finished\n"
+     ]
+    }
+   ],
+   "source": [
+    "for logline in client.get_job_logs(job_name, follow=True):\n",
+    "    print(logline, end='')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cleanup",
+   "metadata": {},
+   "source": [
+    "## Delete the TrainJob\n",
+    "\n",
+    "When the TrainJob is finished, you can delete the resource."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "delete",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# client.delete_job(job_name)"
```

Comment:
If we don't delete job, the container will be in the complete status, right ?

Reply:
We should delete, this shouldn't be commented. Will update.